ts/utils/paymentMethod.ts

```diff
@@ -109,7 +121,8 @@ export const enhanceBancomat = (
     ? abiList[bancomat.info.issuerAbiCode]
     : undefined,
   caption: getTitleFromBancomat(bancomat, abiList),
-  icon: getImageFromPaymentMethod(bancomat)
+  icon: getImageFromPaymentMethod(bancomat),
+  listIcon: pagoBancomatImage
```

Comment:
The `icon` field should represent the icon that in the case of the bancomat should remain the "bancomat" icon and we can avoid to add another field `listIcon`.

Could we move the new bank logo information in `BancomatPaymentMethod.abiInfo.logoUrl`?
All the existing code should be fine after this change.

Reply:
This could be a smarter injection 👍🏼 
I'm going to try....

Reply:
@fabriziofff 
I changed the code as required: now the abi selector gives back an enhanced abi with logoUrl pointing out CDN
In this way if we need an abi logo, that info is available inside abi and not only in the enhanced bancomat (enhanced bancomat includes the abi it self)